kart/tile/tilename_util.py

```diff
@@ -20,3 +23,24 @@ def remove_any_tile_extension(filename):
         if len(filename) != orig_len:
             return filename
     return filename
+
+
+def case_insensitive(name):
+    """
+    Given a name eg "baz", returns a case-insensitive version that can be understood by pathlib.Path.glob
+    or the fnmatch module - eg "[Bb][Aa][Zz]"
```

Comment:
minor: fnmatch is actually case insensitive by default, so I think it makes more sense to just reference `glob` here since its why this function is actually needed

Reply:
Done